osvr/RenderKit/RenderManagerOpenGLATW.h

```diff
@@ -1,825 +0,0 @@
-/** @file
```

Comment:
I think the OpenGLATW pull request was accidentally reverted in this pull request.